configure.ac

```diff
@@ -5830,6 +5830,7 @@ plugin_drbd="no"
 plugin_dpdk="no"
 plugin_entropy="no"
 plugin_ethstat="no"
+plugin_fastpath="yes"
```

Comment:
Since the fastpath plugin depends on `jansson.h`, this variable needs to be enabled only when the library / header are available.